src/crimson/osd/backfill_state.h

```diff
@@ -298,6 +298,8 @@ struct BackfillState::BackfillListener {
     const hobject_t& obj,
     const eversion_t& v) = 0;
 
+  virtual void maybe_flush() = 0;
```

Comment:
Adding a comment re why termed "maybe"?

Reply:
*maybe* means to me the decision whether to flush-or-not belongs to an implementation; the interface doesn't enforce actual *flushing* on each call. I believed it's a common interpretation of such word usage; if you think it's different, I will comment on that.

On the other hand, this extra flexibility isn't currently used. Both implementations are always flushing, so perhaps we should rename to just `flush()`.